rust_dev_preview/cross_service/photo_asset_management/README.md

```diff
@@ -0,0 +1,36 @@
+# Create a photo asset management application with the AWS SDK for Rust
+
+## Overview
+
+The Photo Asset Management (PAM) example app uses Amazon Rekognition to categorize images, which are stored with Amazon S3 Intelligent-Tiering for cost savings. Users can upload new images. Those images are analyzed with label detection and the labels are stored in an Amazon DynamoDB table. Users can later request a bundle of images matching those labels. When images are requested, they will be retrieved from Amazon S3, zipped, and the user is sent a link to the zip.
+
+This Rust implementation shows a number of techniques to run Rust binaries in AWS Lambda. The `src/bin/pam.rs` binary uses the lambda `_HANDLER` environment variable to choose which handler to use for requests. This is a trade off - while it's a single binary in s3, each handler only uses a subset of the features. Because there's a lot of overlap in which features are used, and the entire binary is under 5MB, the convenience of a single Code Asset for CDK outweighs managing multiple binaries _for this application_. Other applications should apply their best judgement for this tradeoff. Individual binaries are available for testing in isolation.
```

Comment:
```suggestion
This Rust implementation shows a number of techniques to run Rust binaries in AWS Lambda. The `src/bin/pam.rs` binary uses the lambda `_HANDLER` environment variable to choose which handler to use for requests. This is a trade off - while it's a single binary in Amazon S3, each handler only uses a subset of the features. Because there's a lot of overlap in which features are used, and the entire binary is under 5MB, the convenience of a single Code Asset for the AWS Cloud Development Kit (AWS CDK) outweighs managing multiple binaries _for this application_. Other applications should apply their best judgment for this tradeoff. Individual binaries are available for testing in isolation.
```